SolarTherm/Systems/Reference_2.mo

```diff
@@ -0,0 +1,538 @@
+within SolarTherm.Systems;
```

Comment:
is this a duplicate of our existing Reference 2 model? if so, let's remove. if not, why not.